tgui-next/packages/tgui/interfaces/MedicalKiosk.js

```diff
@@ -0,0 +1,153 @@
+import { Fragment } from 'inferno';
+import { act } from '../byond';
+import { AnimatedNumber, Box, Button, LabeledList, ProgressBar, Section } from '../components';
+
+export const MedicalKiosk = props => {
+  const { state } = props;
+  const { config, data } = state;
+  const { ref } = config;
+ 
+  return (
+    <Fragment>
+      <Section title="Health Kiosk"
+        textAlign="center">
+        <Box as="span" m={1} textAlign="center">
+          Greetings Valued Employee. Please select your desired treatment type from the options below.
+          <Button
```

Comment:
If you want you can put a `Box with `mt={1}` inbetween button and the line here for a bit of a gap for appearance sake.